settings.json

```
@@ -1665,6 +1665,22 @@
       "requires": []
     }
   ],
+  "typ": [
+    {
+      "command": "typst-lsp",
+      "url": "https://github.com/nvarner/typst-lsp",
+      "description": "Language server for Typst",
+      "requires": []
+    },
```

```suggestion
    }
```
Maybe this will fix?